### ISSUE TITLE
39.  Design Copy Issue (DesignID 153998)

### ISSUE THREAD

**Issue by phstanek** [AquaAerobicSystemsInc/APD-Top-APD-Quick-Wins#39]
DesignID 153998 fails to fully copy, with the pop-up error "Missing the DataColumn 'SBRDesignID' for the SourceColumn 'SBRDesignID':
![image](https://user-images.githubusercontent.com/41307105/48360120-219f3480-e664-11e8-89eb-4438b8f838e4.png)

Attempts to open the copy's CMF tab result in a type conversion error when it tries to lookup the filter header ID:
![image](https://user-images.githubusercontent.com/41307105/48359505-bd2fa580-e662-11e8-9c68-14ca992f31f8.png)

**Comment by phstanek** [comment]
I traced the error back to the _ClonetblPostEqFixedDiffuserInputs_ function, which errors out and prevents the cloth media filter from being fully copied.
It was previously attempting to write to the _tblSBRFixedDiffuserInputs_ table (and failing due to field name differences).  I have updated the code to write to _tblPostEqFixedDiffuserInputs_ and was able to successfully copy design ID 153998 to new design ID 154138 and open the cloth media filter interface.

![image](https://user-images.githubusercontent.com/41307105/48360381-bb66e180-e664-11e8-8be8-34d9c6770576.png)

**Comment by phstanek** [comment]
The same issue was also found and corrected in the ClonetblPreEqFixedDiffuserInputs function and corrected:
![image](https://user-images.githubusercontent.com/41307105/48361366-f9650500-e666-11e8-8edd-159b7d423a2f.png)

I was able to test by successfully copying my test design (ID 152805) to design ID 154140.  It is appending to tblPreEqFixedDiffuserInputs as expected:
![image](https://user-images.githubusercontent.com/41307105/48361497-3cbf7380-e667-11e8-9224-29408c9507f7.png)

**Comment by phstanek** [comment]
The fix has been checked in as Changeset 712 and published to APD Development.

I have tested the update by modifying my test design (152805) to include fixed diffusers in the Pre-EQ. The design was successfully copied to design ID 154144.

tblPreEqFixedDiffuserInputs appears to have been cloned successfully:
![image](https://user-images.githubusercontent.com/41307105/48367370-13f2aa80-e676-11e8-8afa-93902d8ff3ee.png)

I then updated test design 152805 to include fixed diffusers in the Post-EQ.  The design was successfully copied to design ID 154146.

tblPostEqFixedDiffuserInputs appears to have been cloned successfully as well:
![image](https://user-images.githubusercontent.com/41307105/48367626-e9edb800-e676-11e8-9f0d-9165611309fb.png)
